docusaurus/docs/learn/quickstarts/network/help/upgrade-quickstart-network.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Upgrade a Quickstart Network
+id: upgrade-quickstart-network
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This document explains the process for backing up, and upgrading your Ziti Controller and Ziti Router(s).
+
+## Backup Existing Network
+You can certainly back up the entire Ziti Network directory (defaulted to `~/.ziti`). However, if you prefer to only 
+back up the minimum necessary files, here is how to go about doing that.
+1. Create a database backup file by running `ziti edge db snapshot` and copy that file to a backup location. The 
+backup file can be found in the same directory as the Controller's db file which is defaulted to `$ZITI_HOME/db/`. The 
+Controller's config file will also identify where the db is stored by checking the `db` section in the config.
+2. Backup the controller PKI by copying the `$ZITI_HOME/pki/` directory to a backup location
+3. Backup the controller config file, also by copying the file to a backup location
+4. Backup the environment file `$ZITI_HOME/$(hostname -s).env`
+
+## Stop Existing Services
+If your network is using services via `systemd`, stop those services.
+```
+sudo systemctl stop ziti-router ziti-controller
+```
+
+## Get the Latest Binary
+Ziti releases can be found [here](https://github.com/openziti/ziti/releases). However, it is recommended to use the 
+Ziti CLI function `getZiti` to obtain the binary as this will determine the correct architecture and OS as well as 
```

Comment:
I don't care how they get the binary truthfully. I would just tell them there's a helper function and not go so far as to _recommend_ it :)